lib/engine/step/g_1856/track.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative '../base'
```

Comment:
what's the point of this file? should we just delete it?

Reply:
The point of this file is to inject the 1856::Tracker into the default Track logic

Reply:
It looks like I can just put the contents of G1856::Tracker here and have G1856::Track work just the same so I'll do that once I've been able to clean up G1856::Tracker